pkg/lite/lite.go

```diff
@@ -284,12 +285,21 @@ func start(ctx context.Context, opts StartOpts) error {
 	)
 
 	// The devserver embeds the event API.
-	persistenceInterval := consts.LiteDefaultPersistenceInterval
+	pi := consts.LiteDefaultPersistenceInterval
+	persistenceInterval := &pi
+	if opts.RedisURL != "" {
```

Comment:
**thought**: Instead of checking options to detect if it should be backed up, I think a better pattern would be to check the actual configuration of what is running, like `shardedRc.isInMemory`. I know we're passing the raw client, so that may be non-trivial, but this is a suggestion to check for the current set up explicitly rather than infer the current state from a CLI option and the internal logic of another function (`createRedisConnection`).

Reply:
Agree - I'll leave for when we change to `start` and other shapes, as lots can be more explicit then.